pkg/manager/member/tikv_upgrader.go

```diff
@@ -99,7 +99,8 @@ func (u *tikvUpgrader) Upgrade(meta metav1.Object, oldSet *apps.StatefulSet, new
 		i := podOrdinals[_i]
 		store := u.getStoreByOrdinal(meta.GetName(), *status, i)
 		if store == nil {
-			continue
+			setUpgradePartition(newSet, i)
```

Comment:
@DanielZhangQD If not do so, the partition of Statefulset will always be the same as the replicas of component, which will make Statefulset unable to rolling upgrade all the Pods.

Reply:
@cvvz In this case, since no error returned with the change in this PR, so the next Pod will be upgraded in the next sync (30s by default, however, there is something that we need to investigate that even if no errors returned, the tc will be synced in about 10s), right? That's not expected.
What about adding a filed `bootStrapped` in the tikv status and check this field in the different logics? For the code here, if not bootstrapped, just set partition to 0.

Reply:
Ok, that is more efficient